src/app/(proper_react)/redesign/(authenticated)/user/dashboard/View.tsx

```diff
@@ -241,7 +241,7 @@ export const View = (props: Props) => {
 };
 
 // Same logic as breachLogo.js
-function getRandomLightNebulaColor(name: string) {
+export function getRandomLightNebulaColor(name: string) {
```

Comment:
This is `export`ed, but are we using it anywhere? 
I don't see anything in this PR `import`ing it.